src/app/profiles/multipleChoiceEditor.tsx

```diff
@@ -0,0 +1,197 @@
+                                    </CardHeader>
+                                    <CardContent>
+
+                                        <Label>Question Text</Label>
+                                        <Field name="question" type="text" placeholder="" value={questionText} className="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50" onChange={(e: any) => {
+                                            setQuestionText(e.target.value)
+                                            setFieldValue(e.target.name, e.target.value)
+                                        }} />
+
+                                        <DndContext collisionDetection={closestCenter} onDragEnd={onDragEnd}>
+                                            <FieldArray
+                                                name="answers"
+                                                render={arrayHelpers => {
+                                                    ArrayHelperRef.current = arrayHelpers
+                                                    return (
+                                                        <SortableContext items={answers} strategy={verticalListSortingStrategy}>
+                                                            {answers.map((answer, index) => (
+                                                                <SortableInput key={answer.id} answer={answer} index={index} answers={answers} setAnswers={setAnswers} arrayHelpers={arrayHelpers} setFieldValue={setFieldValue} />
+                                                            ))}
+                                                        </SortableContext>
+                                                    )
+                                                }}>
+                                            </FieldArray>
+                                        </DndContext>
+                                    </CardContent>
+                                    <CardFooter className="flex justify-between">
+                                        <Button type="button" onClick={() => {
+                                            setAnswers([...answers, {id: answers.length + 1, label: ''}])
+                                            ArrayHelperRef.current?.push('')
+                                        }}>Add Answer</Button>
+                                        <Button type="submit" onClick={() => {
+                                        }}>Save Question</Button>
+                                    </CardFooter>
+                                </Form>
+                            )}
+                        </Formik>
+                    </Card>
+
+                    {/* VIEWER */}
```

Comment:
if these comments denote different sections, I prefer the pattern of just defining another component like `<Viewer />` / `<Editor />` to improve the readability there. If you use Formik contexts fully this shouldn't be an issue and you won't even need to prop drill